source/module_cusolver/test/mpi_matrix_test.cpp

```diff
@@ -0,0 +1,131 @@
+#include "../mini_pdiag.h"
```

Comment:
If `mini_pdiag` is only used for testing, you can move them to `/test` folder.

Reply:
Ok, thanks for your suggestion!